spec/features/courses/new_spec.rb

```diff
@@ -163,84 +221,84 @@ def save_course
     confirmation_page.save.click
   end
 
-  def select_level(course_creation_params)
-    course_creation_params[:level] = "primary"
+  def select_level(course_creation_params, level_selection:, next_page:)
+    course_creation_params[:level] = level
```

Comment:
I have to say, I'm a bit iffy about the use of a memoized value in a `let` here in the method. It works ... and this is the way the `it` blocks do it, but this is a function and it feels like this should be a parameter.